src/scss/includes/panels/service_panel.scss

```diff
@@ -133,22 +134,19 @@
   }
 }
 
-.service_panel__action__icon {
-  span {
-    font-size: 23px;
-    line-height: 36px;
-  }
-}
-
 @media (max-width: 640px){
 
   .panel.service_panel {
-    height: 215px;
+    height: 235px;
```

Comment:
At first I thought it was just a screenshot artefact but there is a white space at the bottom of the screen. I think this hardcoded height is wrong, as it doesn't take the reduced handle height into account.